### PR TITLE
Add session entity directive: google platform

### PR DESCRIPTION
**Pull request by sloaisiga**

### CODE REVIEW

File: docs/alexa-directives.rst [hunk 1]
```diff
@@ -210,6 +210,129 @@ An APL Command is sent with the `alexaAPLCommand` key in your controller. Just l
 
 
 
+Alexa Presentation Language - T (APLT) Templates
```
Comment (armonge) [inline docs/alexa-directives.rst]: is this correct? seems like it should be part of a separate PR cause it's not related to the google session entities

File: docs/google-assistant-directives.rst [hunk 2]
```diff
@@ -599,3 +599,77 @@ If you need to add output contexts to the dialog flow webhook you can use the `d
...
+      "name": "list-of-fruits"
+    }
+  ]
+
+  // state.js
+
+    app.onIntent("LaunchIntent", {
+        flow: "yield",
+        sayp: "Hello!",
+        sessionEntity: "mySessionEntity",
```
Comment (armonge) [inline docs/google-assistant-directives.rst]: should it be named `dialogflowSessionEntity` ?

File: docs/google-assistant-directives.rst [hunk 3]
```diff
@@ -599,3 +599,77 @@ If you need to add output contexts to the dialog flow webhook you can use the `d
...
+Session Entities
+-------------------------------------------
+
+`Google Documentation <https://cloud.google.com/dialogflow/docs/entities-session>`_
+
+A session represents a conversation between a Dialogflow agent and an end-user. You can create special entities, called session entities during a session. Session entities can extend or replace custom entity types and only exist during the session that they were created for. All session data, including session entities, is stored by Dialogflow for 20 minutes.
+
+For example, if your agent has a @fruit entity type that includes "pear" and "grape", that entity type could be updated to include "apple" or "orange", depending on the information your agent collects from the end-user. The updated entity type would have the "apple" or "orange" entity entry for the rest of the session.
+
+Pre-conditions:
```
Comment (armonge) [inline docs/google-assistant-directives.rst]: this assumes the developers are using voxa-cli

File: test/views.ts [hunk 4]
```diff
@@ -35,6 +35,28 @@ export const views = {
         token: "SkillTemplateToken",
         type: "Alexa.Presentation.APL.ExecuteCommands",
       },
+      APLTCommand: {
```
Comment (armonge) [inline test/views.ts]: let's take all that's not related to session entities into a separate PR

File: docs/google-assistant-directives.rst [hunk 5]
```diff
@@ -599,3 +599,77 @@ If you need to add output contexts to the dialog flow webhook you can use the `d
...
+ 
+ // variables.js
+
+  export function mySessionEntity(voxaEvent: IVoxaEvent) {
+    const sessionEntityType: any = sessionEntity.fruit;
+
+    return sessionEntityType;
+  }
+  
+  // You can create a json file with the entity (mySessionEntity.json)
```
Comment (armonge) [inline docs/google-assistant-directives.rst]: the documentation says to create a `mySessionEntity.json` file, however looking at the tests seems like in reality this is just another key in the views file and not a separate file

**Comment by armonge** [comment]
Also, seems like the integration test is now failling cause there's this new key `sessionEntityTypes` being included on every response, please only included when required. 

https://travis-ci.org/VoxaAI/voxa/jobs/644593791#L2873

File: docs/google-assistant-directives.rst (hunk 2 above)
Comment (sloaisiga) [reply docs/google-assistant-directives.rst]: Key renamed.

File: docs/google-assistant-directives.rst (hunk 5 above)
Comment (sloaisiga) [reply docs/google-assistant-directives.rst]: Updated please verify.

File: test/views.ts (hunk 4 above)
Comment (sloaisiga) [reply test/views.ts]: Updated.

File: docs/google-assistant-directives.rst (hunk 3 above)
Comment (sloaisiga) [reply docs/google-assistant-directives.rst]: Updated.

File: docs/alexa-directives.rst (hunk 1 above)
Comment (sloaisiga) [reply docs/alexa-directives.rst]: Removed from this PR.
